docs/SPEC.md

```diff
@@ -3042,6 +3042,19 @@ Example:
         |> filter(fn:(r) => r._measurement == "net" and r._field == "bytes_sent")
         |> top(n:10, columns:["_value"])
 
+#### Contains 
+
+Tests whether a value is a member of a set.  
+
+Contains has the following parameters: 
+
+| Name    | Type                                          | Description                  |
+| ----    | ----                                          | -----------                  |
+| value   | bool, int, uint, float, string, time          | The value to search for.     |
```

Comment:
Does it work only for these types? Or would it work for any type Flux can express that has an Equal behavior? We haven't really specified equality of flux values though so maybe this is fine for now?

Reply:
I'm not sure that I actually restrict the type in the code...I use val.Equal(Value)  so it might work?  but I only tested for those types. maybe over-deliver and under-promise?